docs/user/quickstart.rst

```diff
@@ -273,6 +273,7 @@ the ``json`` parameter (added in version 2.4.2) and it will be encoded automatic
 
     >>> r = requests.post(url, json=payload)
 
+Note, the ``json`` parameter is ignored if the ``data`` parameter is not empty.
```

Comment:
We should probably note that both `data` *and* `files` take precedence over the `json` parameter.